editor.Rmd

```diff
@@ -100,47 +158,39 @@ Post hashtags at the bottom of each tweet, to make the rest of the content of th
 
 When posting a thread, use each hashtag only once so as not to pollute the timeline of that hashtag.
 
-### Tagging accounts
-
-Only tag the account of an author if they included their username in their author file.
-
-For reviewers, you might search for their account but only tag an account
+### Using emojis 
 
-* it is active,
-* it is not anonymous,
-* it is at least partly used professionally by the owner.
-In case of doubt contact the person whose account you'd like to tag.
-
-For a package wrapping a service present on Twitter you can tag that account (e.g. for a tweet about `rredlist` that accesses IUCN Red List you might tag IUCN Twitter account).
+In tweets, emojis are optional. 
+When using emojis, [do not use too many of them](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible).
 
-When tagging accounts include them in a sentence e.g. "Thanks to `@account1`"/ "As told by `@account2`". 
-We do not use tagging in tweets to ask for attention (i.e. no account names used liked hashtags at the bottom of a tweet).
+### Using gifs
 
-### Emoji use
+In tweets, gifs are optional.
 
-In tweets, emojis are optional. 
-When using emojis, [do not use too many of them](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible).
+* One can include a gif that was created for the post (e.g. an animated plot)
+* But generally not a Giphy gif since it might be interpreted differently by people who know more/less context about the gif in question (e.g. a TV show)
+* When using a gif we should [add their description](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible) by typing [Gif alt: descriptive phrase] at the bottom of the tweet 
+* When tweeting from twitter.com, as of Feb 2020, if you have the "compose image descriptions" setting turned on in your accessibility settings, you can add alt descriptions for gifs as you do for images
```

Comment:
```suggestion
* When tweeting from twitter.com, as of Feb 2020, if you have the "compose image descriptions" setting turned on in your accessibility settings, you can add alt descriptions for gifs as you do for images.
```

Should we write this though as blog editors typically don't have access to the account via Twitter dot com?

I see there's no way to subscribe to https://twitter.com/i/tweetdeck_release_notes, what a pity.

Reply:
> Should we write this though as blog editors typically don't have access to the account via Twitter dot com?

I'm willing to leave some things to be inferred.